.github/workflows/lock-closed.yml

```diff
@@ -0,0 +1,14 @@
+name: Lock issues that are closed and inactive
+
+on:
+  schedule:
+  - cron: 25 * * * *
```

Comment:
Should the cron parameter be quoted? `'25 * * * *'`

Add comment explaining that this will run on the 25th min of each hour?